TODO.txt

```diff
@@ -0,0 +1,68 @@
+TODO C++ Papas Code. Aug  2016
+
+
+FCCSW Guidelines & Code Formatting
+Clear out debugging stuff and commented out sections 
+Ask Joshka: See if there are new tools for automated checking
+ASk Joschka: How best to integrate DOxygen outputs
+
+Commenting
+Still more to do but it has improved.
+
+ROOT: 
+ask Pere re Makefiles
+	Ongoing problem with ROOT and Xcode not working using FCCSW cmakelists
+	Figure out how to get DYLD_LIBRARY_PATH properly sorted for Mac - need advise.
+Tried and lost: ask Père. Look into TWidget. Work out how to stop the display/run without  ctrl-C./Make example close down properly at end (understand what to do with graphics window better)
```

Comment:
are you talking about building a GUI for the display based on root? I think it's not a priority. 
if we want a display with a GUI, we will need to think about the technology we want to use, and plan the effort, possibly involving other people.